tox.ini

```diff
@@ -1,26 +1,26 @@
 [tox]
-envlist = lint, test
+envlist = py36-lint, py36-test
 skipsdist=True
 
 [testenv]
 ignore_errors = False
-basepython = python3.6
 
-[testenv:lint]
+[testenv:py36-lint]
+basepython = python3.6
 use_develop = True
 commands = flake8 --max-line-length 99 --max-complexity 12 nim
 deps = flake8
 
-[testenv:test]
+[testenv:py36-test]
+basepython = python3.6
 use_develop = True
 commands =
-    pytest {posargs}
+    pytest --cov=./ {posargs}
```

Comment:
How does `posargs` work when running `tox` locally?

Reply:
You wrote this, you tell me.

Reply:
Hahaha, my bad. I thought it had to do with `--cov`.